java/Oleksii-Riabov/src/com/zoolatech/lecture4/tasks/_7/Task7.java

```diff
@@ -0,0 +1,45 @@
+package com.zoolatech.lecture4.tasks._7;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Create a method that accepts a string of digits and after each digit
+ * appends its string representation in parentheses.
+ */
+
+public class Task7 {
+    public static void main(String[] args) {
+        String string = "12345678910";
+        System.out.println(stringRepresentation(string));
+    }
+
+    public static String stringRepresentation(String string) {
+        if (string.matches("[0-9]+") && string.length() > 2) {
+            return Stream.of(string)
```

Comment:
you could simplify the processing if you firstly split a string into separate digits, and only then work with it as with a stream of digits

you didn't need to create a stream out of a single string variable

Reply:
Done.